src/aws/cloudfront.tf

```diff
@@ -128,6 +128,30 @@ resource "aws_cloudfront_distribution" "marsha_cloudfront_distribution" {
     viewer_protocol_policy = "redirect-to-https"
   }
 
+  ordered_cache_behavior {
+    path_pattern     = "*/depositedfile/*"
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = local.s3_destination_origin_id
+    trusted_signers  = []
+    trusted_key_groups = [aws_cloudfront_key_group.marsha_cloudfront_signer_key_group.id]
+
+    forwarded_values {
+      query_string = true
+      headers = ["Access-Control-Request-Headers", "Access-Control-Request-Method", "Origin"]
+
+      cookies {
+        forward = "none"
+      }
+    }
+
+    min_ttl                = 0
+    default_ttl            = 3600
+    max_ttl                = 86400
+    compress               = true
+    viewer_protocol_policy = "redirect-to-https"
+  }
```

Comment:
This comment is not related to this line.

In the lambda.tf file, the convert lambda must be edited to allow the ephemeral storage configuration: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function#lambda-ephemeral-storage

Something like this should be added:

```
ephemeral_storage {
    size = "{var.lambda_convert_memory}"
  }
```